spec/Section 4 -- Introspection.md

```diff
@@ -331,7 +334,12 @@ Fields
 * `kind` must return `__TypeKind.INPUT_OBJECT`.
 * `name` must return a String.
 * `description` may return a String or {null}.
+* `requiresExactlyOneField` must return a Boolean
 * `inputFields`: a list of `InputValue`.
+  * if `requiresExactlyOneField` is {true}:
```

Comment:
In other places we have "Type Validation" sub-sections. That would be a better place for this list set